.github/workflows/test-lang-ruby.yml

```diff
@@ -76,10 +77,11 @@ jobs:
         - '2.5'
         - '2.6'
         - '2.7'
+        - '3.0'
     steps:
       - uses: actions/checkout@v2
 
-      - uses: actions/setup-ruby@v1
+      - uses: ruby/setup-ruby@v1
```

Comment:
If the build was able to use `ruby/setup-ruby` successfully, does that mean that it was already approved?

Reply:
Opened https://issues.apache.org/jira/browse/INFRA-21366 to ask the question

Reply:
> If the build was able to use ruby/setup-ruby successfully, does that mean that it was already approved?

No, I don't believe it does. IIUC this build ran on your fork in your account, which isn't restricted by Apache rules.

But since you got the official approval from infra we are good to go.